tests/test_runpath.py

```diff
@@ -0,0 +1,31 @@
+import nose
+import os
+import shutil
+import tempfile
+
+import cle
+
+TEST_BASE = os.path.join(os.path.dirname(os.path.realpath(__file__)),
+                                         os.path.join('..', '..', 'binaries'))
+
+def test_runpath():
+    tempdir = tempfile.mkdtemp()
+    try:
+        runpath_file = os.path.join(TEST_BASE, 'tests', 'x86_64', 'runpath')
```

Comment:
This needs a new binary I will publish in a minute.

Reply:
see https://github.com/angr/binaries/pull/24/commits/6d110bac9211e86ea65fdfd05fc1f31e2f0d9927